libstorage/Table.cpp

```diff
@@ -231,6 +231,22 @@ void Condition::limit(size_t offset, size_t count)
 {
     m_offset = offset;
     m_count = count;
+    m_dirty = true;
+}
+
+size_t Condition::getOffset()
```

Comment:
add  `getOffset()/getCount()/isDirty()` is necessary?

Reply:
remove it.